xm-commons-search/src/main/java/com/icthh/xm/commons/search/builder/ExistsQueryBuilder.java

```diff
@@ -0,0 +1,25 @@
+package com.icthh.xm.commons.search.builder;
```

Comment:
add java doc for all classes that was added. It should contains reference to native library (take a look on another classes)

Reply:
done